### ISSUE TITLE
Dask Docs Accessibility Sprint 2021-12-2

### ISSUE THREAD

**Issue by ian-r-rose** [dask/community#208]
Hi all!

As part of our regular community meeting (cf #207) we'd like to try something a bit different. After our regular updates, we're going to do a ~1hr sprint on making the Dask documentation more accessible to screen readers (see some more discussion in #193). This is a model that's been led to great success by some of our colleagues at Quansight, @tonyfast and @isabela-pf in some of our peer projects, including Jupyter, NumPy, and Scikit-learn.

I encourage people in the Dask community to participate, as well as invite other people in their networks.

Some documentation on the sprint has been prepared by @tonyfast [here](https://hackmd.io/xYRkmgi1S9SpQxFXY8q1Ig?view). Participants shouldn't need to prepare much, but the interested can get a head start by reading some of the resources @isabela-pf has prepared here: https://github.com/dask/community/issues/193#issuecomment-975829431 .

**Comment by scharlottej13** [comment]
[Here](https://github.com/scharlottej13/dask/pull/1) is the PR we'll use!

**Comment by GenevieveBuckley** [comment]
If anyone gets really enthusiastic, people could potentially also look at the Dask blogposts for images missing alt-text there as well. Or maybe it could be good for a future accessibility sprint?

There's a mix of markdown syntax `![alt text](/images/file.png)` and html syntax `<img src="/images/file.png" alt="alt text">` used for images in this repository.

- Website: https://blog.dask.org/
- Github repository: https://github.com/dask/dask-blog

**Comment by ian-r-rose** [comment]
Closing this as the sprint has concluded and the PR is in, thanks all!

Follow-up sprints (which I'd like to do) will get separate issues and messaging
